### ISSUE TITLE
Support Deno 2

### ISSUE THREAD

**Issue by injust** [hasundue/molt-action#33]
Deno 2 appears to work with the lockfile disabled.

With the lockfile enabled, I get this error:

```
Error: Unsupported lockfile version '4'. Try upgrading Deno or recreating the lockfile.
Error: Process completed with exit code 1.
```

**Comment by hasundue** [comment]
Okay let's do it since Deno 2.0 doesn't include `deno update` for some reason.

**Comment by hasundue** [comment]
Well it's coming soon actually https://github.com/denoland/deno/pull/26219

**Comment by Wissididom** [comment]
It now apparently is in v2.1.0 of Deno. See this merged PR: https://redirect.github.com/denoland/deno/pull/26942

**Comment by hasundue** [comment]
Yes it is. I've just tried the feature and it seems good enough for me to deprecate molt and molt-action. 

Some features that molt has are missing in the Deno CLI but I'd rather consider contributing to Deno it self than making my own wheel.

Regarding GitHub Action, someone will probably make a good one using the CLI or already did hopefully. If not I might be making one with a similar interface to this.

**Comment by landsman** [comment]
What about automated PRs?

**Comment by hasundue** [comment]
No GitHub action for `deno outdated` from denoland or the community yet?
I've been away from deno stuff for a while so...
